QUEST.tsv

```diff
@@ -522,210 +522,210 @@ QUEST_20150317_000521	Seems to be asleep.
 QUEST_20150317_000522	Enraged Owl Statue
 QUEST_20150317_000523	No responses.
 QUEST_20150317_000524	Regretful Merchant's Soul
-QUEST_20150317_000525	If you don't have money, you're no different than a dead man.
-QUEST_20150317_000526	I should've listened to my wife. {nl}She said it's bad luck to be friends with scholars… {nl}I think I'm really going out of business!
+QUEST_20150317_000525	If you don't have money, you're no different from a dead man.
+QUEST_20150317_000526	I should've listened to my wife. {nl}She says it's bad luck to be friends with scholars… {nl}I'm really going out of business!
```

Comment:
I think the past tense was ok here? The wife stated it was bad luck. It's not like she's saying it now.